Bio/SwissProt/__init__.py

```diff
@@ -406,7 +406,7 @@ def _read(handle):
             # **HA SAM; Annotated by PicoHamap 1.88; MF_01138.1; 09-NOV-2003.
             pass
         else:
-            raise SwissProtParserError(f"Unknown keyword '{key}' found", line=line)
+            raise SwissProtParserError(f"Unknown keyword {key!r} found", line=line)
```

Comment:
i like the version with quotes better.
Also, I think `!r` is not needed here.

Reply:
If the problematic variable were to have tabs (etc) in it, then the repr version is better, but in many cases yes there is little added value.

I only changed this because my copy of flake8 flagged it.

Reply:
That is confusing, but no:

```pycon
>>> key = "very\tproblematic"
>>> print(key)
very	problematic
>>> print(f"Unknown keyword '{key}' found")
Unknown keyword 'very	problematic' found
>>> print(f"Unknown keyword {key!r} found")
Unknown keyword 'very\tproblematic' found
```

Reply:
OK then.